GitHubWorkflow.md

```diff
@@ -0,0 +1,33 @@
+# GitHub workflow
+Samenwerking in GitHub kan erg complex zijn en leiden tot ongewenste effecten zoals ingewikkelde bestandsconflicten en  verlies van aangebrachte wijzigingen.
+Om dat tegen te gaan is het essentieel goede afspraken te maken over deze samenwerking. 
+Binnen de HaalCentraal BRK Bevragen repositories gebruiken we daarvoor Gitflow.
+
+## Gitflow
+Binnen Gitflow wordt de 'master' branch alleen gebruikt om de officiële releases van een product op te slaan waarbij we de diverse releases met behulp van een 'tag' van elkaar onderscheiden.
+De 'master' branch wordt dus niet gebruikt om tussentijdse wijzigingen vast te leggen teneinde stukjes functionaliteit toe te voegen.
+
+Voor het voorbereiden van een nieuwe release en in het kader van het stap voor stap uitbreiden van de functionaliteit worden de zogenaamde  'develop' branch en 'feature' branches gebruikt.
+De 'feature' branches worden gebruikt voor het dedicated werken aan een specifieke hoeveelheid functionaliteit en de 'develop' branch voor het samenvoegen van de in de 'feature' branches gebouwde code met de laatste officiële versie.
+Bij het bepalen van wat het doel van een PR is het dus van belang te weten wat de bedoeling van een PR is. 
+Gaat het om een tussentijdse aanpassing van een feature die daarmee echter nog niet gereed is dan betreft het een PR op de gerelateerde 'feature' branch.
+Gaat het om een feature dat gereed is om samengevoegd te worden met de toekomstige versie van het product dan is het doel de 'develop' branch.
+
+Of en zo ja wanneer een 'feature' branch met de 'develop' branch wordt samengevoegd is aan het HaalCentraal team.
+Zo kan besloten worden een feature niet met de komende release aan te bieden maar met een release die verder in de toekomst ligt.
+Een 'feature' branch die eenmaal is samengevoegd zal daarna komen te vervallen zodat uit de lijst met branches duidelijk wordt welke functionaliteit nog niet ondergebracht is in een release.
+
+Zodra de inhoud van de 'develop' branch naar de mening van het HaalCentraal team gereed is voor een officiële versie wordt deze doorgezet naar een zogenaamde 'release' branch.
```

Comment:
Wij kiezen er vooralsnog voor om niet met een release branche te werken. Ik weet ook niet of we dat gaan doen. Lijkt mij dan ook dat je dat deel van de beschrijving niet moet opnemen zolang we niet zo werken.